spacewalk/config/spacewalk-config.spec

```diff
@@ -136,6 +140,12 @@ sysconf_addword /etc/sysconfig/apache2 APACHE_SERVER_FLAGS SSL
 sysconf_addword /etc/sysconfig/apache2 APACHE_SERVER_FLAGS ISSUSE
 %endif
 
+if [ -f /etc/apache2/vhosts.d/vhost-ssl.conf ]; then
```

Comment:
I think we can use this logic to update the DocRoot, which is your goal. If the file exists, then we can check if the value is the one we want, otherwise, we replace it.
If the file exists and has the wanted value, let it stay.

On top of this if we update the tool  `spacewalk-setup-httpd`, newly generated files will have the new location.
Calling the tool automatically can be more tricky since we don't know which parameters where used.